### ISSUE TITLE
Upgrade from `C120` -> `C121` on `Win x64` via `1.63.x - Nightly` (Checking C121)

### ISSUE THREAD

**Issue by kjozwiak** [brave/brave-browser#35281]
As per process, QA runs through the following cases to ensure that the major chromium bump that's about to get merged into `master` doesn't regress the `Nightly` channel. Once the major chromium bump is merged into `master`, QA will run through a full manual pass on `Windows` & `Android`.

### Startup & Components

- [x] Verify that Brave is only contacting `*.brave.com` endpoints on first launch using either `Charles Proxy`, `Fiddler`, `Wireshark` or `LittleSnitch` (or a similar application)
  - [x] Verify that opening a NTP doesn't trigger any outbound connections related to widgets without user interaction
- [x] Remove the following component folders and ensure that they're being re-downloaded after restarting the browser:
  - [x] `afalakplffnnnlkncjhbmahjfjhmlkal`: `AutoplayWhitelist.dat`, `ExtensionWhitelist.dat`, `ReferrerWhitelist.json` and `Greaselion.json`
  - [x] `CertificateRevocation`
  - [x] `gccbbckogglekeggclmmekihdgdpdgoe`: (Sponsored New Tab Images)
  - [x] `gkboaolpopklhgplhaaiboijnklogmbc`: Brave Ad Block List Catalog
  - [x] `iodkpdagapdfkphljnddpjlldadblomo`: Brave Ad Block Updater
  - [x] `mfddibmblmbccpadfndgakiopmmhebop`: Brave Ad Block List Catalog
  - [x] `oofiananboodjbbmdelgdommihjbkfag`: HTTPSE
  - [x] `Safe Browsing`
- [x] Restart the browser, load `brave://components`, wait for 8 mins and verify that no component shows any errors

**Note:** Always double check `brave://components` to make sure there's no errors/missing version numbers

### Upgrade

- [x] Make sure that data from the last version appears in the new version OK
- [x] Ensure that `brave://version` lists the expected Brave & Chromium versions
- [x] With data from the last version, verify that:
  - [x] Bookmarks on the bookmark toolbar and bookmark folders can be opened
  - [x] Cookies are preserved
  - [x] Installed extensions are retained and work correctly
  - [x] Opened tabs can be reloaded
  - [x] Stored passwords are preserved
  - [x] Sync chain created in previous version is retained
  - [x] Social media-blocking buttons changes are retained
  - [x] Custom filters under brave://settings/shields/filters are retained
  - [x] Custom lists under brave://settings/shields/filters are retained
  - [x] Rewards
    - [x] BAT balance is retained
    - [x] Auto-contribute list is retained
    - [x] Both Tips and Monthly Contributions are retained
    - [x] Panel transactions list is retained
    - [x] Changes to rewards settings are retained
    - [x] Ensure that Auto Contribute is not being enabled when upgrading to a new version if AC was disabled
  - [x] Ads
    - [x] Both `Estimated pending rewards` & `Ad notifications received this month` are retained
    - [x] Changes to ads settings are retained
    - [x] Ensure that ads are not being enabled when upgrading to a new version if they were disabled
    - [x] Ensure that ads are not disabled when upgrading to a new version if they were enabled

**Comment by MadhaviSeelam** [comment]
Manual pass verification complete 
```
Brave | 1.63.110 Chromium: 121.0.6167.57 (Official Build) nightly (64-bit)
-- | --
Revision | f7a7c2ee42626f5a1310c9bd6789d0438e78ca6e
OS | Windows 11 Version 22H2 (Build 22621.3007)
```

**Comment by MadhaviSeelam** [comment]
Verification `In progress` using
```
Brave | 1.63.110 Chromium: 121.0.6167.57 (Official Build) nightly (64-bit)
-- | --
Revision | f7a7c2ee42626f5a1310c9bd6789d0438e78ca6e
OS | Windows 11 Version 22H2 (Build 22621.3007)
```



<details>
<summary>C120 --> C121 on-demand upgrade check - PASSED</summary>

* installed `1.63.113 Chromium: 120.0.6099.217` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `nightly` (usually `cr-bump-update-tests-x64` is used but this time around `nightly` was used)
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.63.110 Chromium: 121.0.6167.57` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`**
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/98358127/2db8e6f2-0273-43f6-b63c-28ed37912233)
![image](https://github.com/brave/brave-browser/assets/98358127/94f26470-0786-40de-bb53-4a8e61fd6b41)
![image](https://github.com/brave/brave-browser/assets/98358127/456be6c6-4429-4daf-8525-9643156e1817)
![image](https://github.com/brave/brave-browser/assets/98358127/77f57d21-f42a-4fbc-99e0-1cebe15d3d6c)

**Comment by kjozwiak** [comment]
Verification **PASSED** on `Win 10 x86` using the following build(s):

```
Brave | 1.63.113 Chromium: 121.0.6167.57 (Official Build) nightly (32-bit)
-- | --
Revision | baf10610d0ea5f609e5bf2e656f91c2e58568fd9
OS | Windows 10 Version 22H2 (Build 19045.3803)
```

<details>
<summary>C120 --> C121 on-demand upgrade check - PASSED</summary>

* installed `1.63.101 Chromium: 120.0.6099.199` and closed Brave
* created `HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\UpdateDev\url` and set `url` as the following:
  * `https://updates.brave.com/service/update2`
* set `Computer\HKEY_LOCAL_MACHINE\SOFTWARE\BraveSoftware\Update\ClientState\{C6CB981E-DB30-4876-8639-109F8933582C}\ap` as `cr-bump-update-tests-x86` (usually `cr-bump-update-tests-x86` is used but this time around `nightly` was used)
* realunched Brave and visited `brave://settings/help` and ensured that new update was being pulled/downloaded
* ensured that clicking `Relaunch` restarted the browser without any issues
* ensured Brave was updated to `1.63.113 Chromium: 121.0.6167.57` without any issues and you can restart the browser

**`Example`** | **`Example`** | **`Example`** | **`Example`** 
----------------|----------------|----------------|---------------
![image](https://github.com/brave/brave-browser/assets/2602313/055dc9c4-573f-4f92-b366-5ef3ae79ea66) | ![image](https://github.com/brave/brave-browser/assets/2602313/0151751c-e6f2-4455-b1af-413e10d4a56d) | ![image](https://github.com/brave/brave-browser/assets/2602313/d90eb4be-2e36-4289-8b8d-39b645c6348a) | ![image](https://github.com/brave/brave-browser/assets/2602313/55ff04e9-71c2-4dff-85f7-0cb9279a650a)
</details>
